website_customer_also_purchased/views/layout.xml

```diff
@@ -0,0 +1,12 @@
+<openerp>
+    <data>
+		<template id="assets_frontend" inherit_id="website.assets_frontend" name="Shop Stock State">
+  			<xpath expr="." position="inside">
+                <link rel='stylesheet' href='/website_customer_also_purchased/static/src/js/node_modules/slick-carousel/slick/slick.less'/>
```

Comment:
why using a external library?, bootstrapp already has an implementation of carousel http://bootsnipp.com/snippets/featured/thumbnail-carousel